release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -885,6 +904,24 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 [id="ocp-4-10-builds-bug-fixes"]
 ==== Builds
 
+* Before this update, if you created a build configuration containing an image change trigger in {product-title} 4.7.x or earlier, the image change trigger might trigger builds continuously.
++
+This issue happened because, with the deprecation and removal of the `lastTriggeredImageID` field from the `BuildConfig` spec for Builds, the image change trigger controller stopped checking that field before instantiating builds. {product-title} 4.8 introduced new fields in the status that the image change trigger controller needed to check, but didn't.
```

Comment:
```suggestion
This issue happened because, with the deprecation and removal of the `lastTriggeredImageID` field from the `BuildConfig` spec for Builds, the image change trigger controller stopped checking that field before instantiating builds. {product-title} 4.8 introduced new fields in the status that the image change trigger controller needed to check, but did not.
```

Reply:
Updated. Thanks.